{"ID": "491f2639-03e8-446a-834a-078a2cfc2162", "effective_cve": "CVE-2020-26116", "image": {"exact": "docker.io/anchore/test_images@sha256:347fba6fbfa15d4e11217f9d49bf70a5a6eef35c6c642dc8c5db89115912d0c1"}, "label": "TP", "package": {"name": "python2.7-minimal", "version": "2.7.16-2+deb10u1"}, "timestamp": "2024-05-16T17:54:53.999305+00:00", "tool": "grype@v0.77.4", "user": "wagoodman", "vulnerability_id": "CVE-2020-26116"}